gym_compete/policy.py

```diff
@@ -212,19 +213,20 @@ def loop_fn(time, cell_output, cell_state, loop_state):
                     last_out = seq_to_batch(last_out)
                     self.state_out.append(final_state)
 
-                    return last_out
+                    return last_out, input_seq
 
-                value_out = lstm(0, 'v')
+                value_out, value_ff_acts = lstm(0, 'v')
                 self.value_fn = tf.contrib.layers.fully_connected(value_out, 1, activation_fn=None)
                 if self.normalized and self.normalized != 'ob':
                     self.value_fn = self.value_fn * self.ret_rms.std + self.ret_rms.mean  # raw = not standardized
 
-                mean = lstm(2, 'p')
+                mean, policy_ff_acts = lstm(2, 'p')
                 mean = tf.contrib.layers.fully_connected(mean, ac_space.shape[0],
                                                          activation_fn=None)
                 logstd = tf.get_variable(name="logstd", shape=[1, ac_space.shape[0]],
                                          initializer=tf.zeros_initializer())
 
+                self.ff_out = [value_ff_acts, policy_ff_acts]
```

Comment:
Think it'd be good to have difference instance variables (or a dict with keys) for each of value and policy activations, as as we often might only want to look at one of them.